superset/assets/src/components/TableSelector.jsx

```diff
@@ -370,10 +370,12 @@ export default class TableSelector extends React.PureComponent {
       <div className="section">
         <ControlLabel>
           {t('See table schema')}{' '}
-          <small>
-            ({this.state.tableOptions.length} {t('in')}{' '}
-            <i>{this.props.schema}</i>)
-          </small>
+          {this.props.schema && (
+            <small>
+              ({this.state.tableOptions.length} {t('in')}{' '}
+              <i>{this.props.schema}</i>)
```

Comment:
And also remove the close paren here

```suggestion
              <i>{this.props.schema}</i>
```

Reply:
@etr2460 this is needed for the `(<n> in <schema>)`.